tests/python/PyImportSortBearTest.py

```diff
@@ -21,3 +25,20 @@
      "import abc\nimport xyz\n"),
     settings={"known_standard_library_imports": "xyz",
               "known_first_party_imports": "abc"})
+
+
+class PyImportSortBearTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.uut = PyImportSortBear(Section('name'), Queue())
+
+    def test_run(self):
+        file = "from abc import xyz, fgh"
+        self.check_results(
+            self.uut,
+            ["from abc import xyz, fgh"],
+            ["Imports fgh xyz are sorted incorrectly."], file)
+        self.check_results(
+            self.uut,
+            ["import z", "import g", "import e"],
+            ["Imports e g are sorted incorrectly."], file)
```

Comment:
Umm, the statement will always be shown when the bear finds an error right?. If they are sorted correctly then the bear should not find out any error at all?. Could you please suggest me something?.